docs/developers/v2/fork-yearn-ui.md

```diff
@@ -0,0 +1,229 @@
+# Fork Yearn UI
+
+This is a step-by-step guide on how to fork and customize [Macarena Finance](https://github.com/yearn/macarena-finance) which is a yearn UI made to be forked. Deploying your own UI makes you eligible to receive partner [profit-sharing](https://docs.yearn.finance/partners/introduction#profit-share-model) fees.
+
+*Table of Conent*
+- [Install required software](#install-required-software)
+- [Clone Macarena Finance](#clone-macarena-finance)
+- [Change Vaults Filter](#change-vaults-filter)
+- [Change Logo](#change-logo)
+- [Change Colors](#change-colors)
+- [Change SEO settings](#change-seo-settings)
+- [Apply to receive partner fees](#apply-to-receive-partner-fees)
```

Comment:
No need to have a table of contents. Docusaurus generates it on the left automatically